src/cli/install.sh

```diff
@@ -117,7 +117,7 @@ if test $(basename $SHELL) == "fish"; then
     IS_BUN_AUTO_UPDATE="true" SHELL="fish" $exe completions >/dev/null 2>&1
     if test -f $HOME/.config/fish/config.fish; then
         echo -e "\n# bun\nset -Ux BUN_INSTALL \"$bun_install\"" >>"$HOME/.config/fish/config.fish"
-        echo -e "set -px --path PATH \"$bin_dir\"\n" >>"$HOME/.config/fish/config.fish"
+        echo -e "fish_add_path \"$bin_dir\"\n" >>"$HOME/.config/fish/config.fish"
```

Comment:
Note that these two changes are *not* quite equivalent.

By default, `fish_add_path` modifies `$fish_user_paths`, a universal variable. If you just want to modify `$PATH` by default, this should be done with `fish_add_path -P …`.